Packs/Silverfort/Integrations/Silverfort/Silverfort_description.md

```diff
@@ -1,8 +1,10 @@
 ## Silverfort Integration
-- To use this integration you need to enter the **Server URL** and **API Key** parameters. 
-- To generate an API token:
-    1. From the Silverfort Admin Console, navigate to **Settings > Advanced**.
-    2. In the Authentication Tokens section, click **Generate Token**.
-    3. Copy the generated token and save it in a safe place.
+- To use this integration you need to enter the **Server URL** ,**API Key** parameters. 
+- To create **API Key**:
+    1. On the Silverfort Admin Console, navigate to the **SETTINGS** page, and then select **Silverfort API**.
+    2. At the Application **User ID** field, click **Create App User**.
+    3. Copy the values of the **App User ID** and **App User Secret**.
+    4. Save.
+    5. The API KEY is "**App USer ID**:**App USer Secret**".
```

Comment:
```suggestion
    5. The API KEY is "**App User ID**:**App User Secret**".
```
should we explicitly state the delimiter? (something like, "mind the `:` between User ID and Secret")

Reply:
Sure, if you believe it'll help clarify the ID/secret.